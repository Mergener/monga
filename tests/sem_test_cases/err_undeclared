type T = {
	a : int;
	b : float;
};

function main() {
	var i : int;

	i = x;
}