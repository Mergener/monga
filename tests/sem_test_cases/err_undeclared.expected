[Error] Module tests/sem_test_cases/err_undeclared, line 9, column 8: Variable 'x' was not declared in this scope.
Semantic errors were caught during analysis.
